### PR TITLE
Create UI for forgot password sate

### PR DESCRIPTION
**Pull request by DongTrieu1998**

### CODE REVIEW

**Comment by DongTrieu1998** [comment]
Screenshot 2024-09-18 001006

@thanhdat369

File: fudy/SignInScreen/ForgotPasswordPanel.qml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+		InputBox {
+			id: retypePasswordInput
+
+			Layout.alignment: Qt.AlignRight
+			title: qsTr("Hint")
+			placeHolder: qsTr("Enter your hint")
+		}
+
+		FudyButton {
+
```
Comment (ghost) [inline fudy/SignInScreen/ForgotPasswordPanel.qml]: -1

File: fudy/SignInScreen/ForgotPasswordPanel.qml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	Text {
+		id: titleContent
+
+		text: qsTr("Please enter your details")
+		font: FudyFont.heading2
+	}
+
+	ColumnLayout {
+
```
Comment (ghost) [inline fudy/SignInScreen/ForgotPasswordPanel.qml]: -1

**Review by ghost — APPROVED** [review]

File: fudy/SignInScreen/ForgotPasswordPanel.qml (hunk 1 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/ForgotPasswordPanel.qml]: this line was removed

File: fudy/SignInScreen/ForgotPasswordPanel.qml (hunk 2 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/ForgotPasswordPanel.qml]: this line was removed

**Pull request merged**